### ISSUE TITLE
CreateAccountExample fails on `mainnet` with status `InvalidSignature`

### ISSUE THREAD

**Issue by ericleponner** [hashgraph/hedera-sdk-swift#36]
### Description

Example `CreateAccount` works fine on `previewnet` and `testnet` but fails on `mainnet`.
The following output is printed by the example:

```
Swift/ErrorType.swift:200: Fatal error: Error raised at top level: transaction `0.0.1044508@1676330085.621415178` failed pre-check with status `InvalidSignature`
2023-02-14 00:14:52.256990+0100 CreateAccountExample[23301:271003] Swift/ErrorType.swift:200: Fatal error: Error raised at top level: transaction `0.0.1044508@1676330085.621415178` failed pre-check with status `InvalidSignature`
```

### Steps to reproduce

1) Setup `OPERATOR_ACCOUNT_ID` and `OPERATOR_KEY` with a `mainnet` account
2) Run `Examples/CreateAccount/main.swift`
=> creation aborts and prints output described above.

### Hedera network

mainnet

### Version

v0.12.0

### Operating system

macOS

**Comment by SimiHunjan** [comment]
@ericleponner confirming you updated to use a mainnet account ID and key generated from a wallet or another source when trying to sign for the transaction?

cc: @izik1

**Comment by SimiHunjan** [comment]
Just read the body of the issue and looks like you did!

**Comment by ericleponner** [comment]
Yes. I have an account on `mainnet` generated by Blade Wallet.

**Comment by mehcode** [comment]
I believe this is fixed in 0.13.0. Please re-open if you are experiencing the same issue.

**Comment by ericleponner** [comment]
I confirm that this is fixed in 0.13.0.
